gripmock/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# BASE_TAG *may* include VERSION, docker docs use TAG for the image being built
+# https://docs.docker.com/engine/reference/commandline/image_tag/
+ARG BASE_TAG=latest
+
+FROM tkpd/gripmock:v1.12.1
+
+# NOTE: the protoc stuff is copied from the builder image repo.
+
+# protoc command
+ENV PROTOC_VERSION=24.3
+RUN PROTOC_ARCH=$(uname -m); \
+  if test "$PROTOC_ARCH" = "aarch64"; then PROTOC_ARCH=aarch_64; fi; \
+  wget https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-${PROTOC_ARCH}.zip \
+  && unzip -q -o protoc-${PROTOC_VERSION}-linux-${PROTOC_ARCH}.zip -d /usr/local bin/protoc \
+  && chmod +x /usr/local/bin/protoc \
+  && unzip -q -o protoc-${PROTOC_VERSION}-linux-${PROTOC_ARCH}.zip -d /usr/local include/* \
+  && chmod -R +rx /usr/local/include \
+  && rm -rf protoc-${PROTOC_VERSION}-linux-${PROTOC_ARCH}.zip
+
+# Download common google api proto files.
+ENV GOOGLE_COMMON_PROTOS_VERSION=1.50.0
```

Comment:
These are present in the proto-include.tar.gz included on every release. the runes to install both the google headers and the grpc onese are just

rm -rf proto-include
curl -fsSOL https://github.com/datatrails/go-datatrails-common-api/releases/download/{{.DATATRAILS_COMMON_API_VERSION}}/proto-include.tar.gz
tar -zxf proto-include.tar.gz
rm proto-include.tar.gz

Reply:
The avid taskfiles/Taskfile_apis.yml bootstrap step is a good reference